lib/faker.js

```diff
@@ -94,18 +137,46 @@ function createFakerSchema(schema) {
       mutateWithFakerProperty(s.properties.lon, 'location.longitude')
       mutateWithFakerProperty(s.properties.lat, 'location.latitude')
 
-      mutateWithFakerProperty(
-        s.properties.attachments.items.properties.driveDiscoveryId,
-        'mapeo.id',
-      )
-      mutateWithFakerProperty(
-        s.properties.attachments.items.properties.hash,
-        'mapeo.id',
-      )
-      mutateWithFakerProperty(
-        s.properties.attachments.items.properties.name,
-        'mapeo.id8',
-      )
+      // Attachments
+      for (const attachmentDef of s.definitions.attachment.oneOf) {
+        mutateWithFakerProperty(
+          attachmentDef.properties.driveDiscoveryId,
+          'mapeo.id',
+        )
+        mutateWithFakerProperty(attachmentDef.properties.hash, 'mapeo.id')
+        mutateWithFakerProperty(attachmentDef.properties.name, 'mapeo.id8')
+        mutateWithFakerProperty(
+          attachmentDef.properties.driveDiscoveryId,
+          'mapeo.id',
+        )
+        mutateWithFakerProperty(attachmentDef.properties.hash, 'mapeo.id')
+        mutateWithFakerProperty(attachmentDef.properties.name, 'mapeo.id8')
+
+        // TODO: Can fake more of the fields
```

Comment:
didn't want to get carried away with it. it's kind of tedious and we don't have a good way of generating "dependent" values i.e. `GPSLatitudeRef` is usually based on `GPSLatitude` (although I guess anything can go against the spec and put whatever values wherever 😄 )